src/Microsoft.FileFormats/PE/PEStructures.cs

```diff
@@ -99,6 +99,19 @@ public enum ImageFileMachine
         Arm64       = 0xAA64
     }
 
+    /// <summary>
+    /// Characteristics (IMAGE_FILE)
+    /// </summary>
+    [Flags]
+    public enum ImageFile : ushort
+    {
+        RELOCS_STRIPPED         = 0x0001,
```

Comment:
Nit: We've typically used Pascal casing following BCL coding style rules

Reply:
I was being lazy and copied this enum from clrmd. I'll fix it.